language/de/acp/profile.php

```diff
@@ -164,8 +164,8 @@
 	'STEP_2_EXPLAIN_EDIT'			=> 'Hier kannst du einige Einstellungen für die Feldart vornehmen.<br /><strong>Bitte beachte, dass Änderungen keine Auswirkungen auf die bereits existierenden Angaben deiner Benutzer haben.</strong>',
 	'STEP_2_TITLE_CREATE'			=> 'Spezifische Optionen der Feldart',
 	'STEP_2_TITLE_EDIT'				=> 'Spezifische Optionen der Feldart',
-	'STEP_3_EXPLAIN_CREATE'			=> 'Da du mehr als ein Sprachpaket installiert hast, musst du auch Angaben für die anderen Sprachen machen. Du kannst diese Angaben auch später ergänzen, das Feld arbeit derweil mit der Standardsprache.',
-	'STEP_3_EXPLAIN_EDIT'			=> 'Da du mehr als ein Sprachpaket installiert hast, kannst du hier die Angaben für die anderen Sprachen ändern oder ergänzen. Wenn keine Angaben gemacht werden, arbeitet das Feld mit der Standardsprache.',
+	'STEP_3_EXPLAIN_CREATE'			=> 'Da du mehr als ein Sprachpaket installiert hast, musst du auch Angaben für die anderen Sprachen machen. Wenn keine Angaben gemacht werden, verwendet benutzerdefinierte Profilfeld derweil die Standardsprache, du kannst diese Angaben auch später ergänzen.',
```

Comment:
Ich würde da näher an der Ursprungsversion bleiben:

```suggestion
	'STEP_3_EXPLAIN_CREATE'			=> 'Da du mehr als ein Sprachpaket installiert hast, musst du auch Angaben für die anderen Sprachen machen. Wenn du keine Angaben machst, werden für dieses benutzerdefinierte Profilfeld die Einstellungen für die Standardsprache verwendet. Du kannst diese Angaben auch später ergänzen.',
```